source/localizable/feedspec/validator.html.md.erb

```diff
@@ -0,0 +1,9 @@
+---
+parent: feedspec
+title: validator
+---
+
+# Data Feed Validator <%= edit_link %>
+
+You can validate your XML feed according to <%= settings.site_name.capitalize %> standards
+using our public <a href="http://<%= settings.feed_validator_domain %>">Data Feed Validator</a>.
```

Comment:
`link_to` is your friend. Also I'm not sure we want a whole page to act as a "redirect". It's kinda pointless.